### PR TITLE
Blog - Node-Red Builder a GPT (Alpha) by FlowFuse

### PR DESCRIPTION
**Pull request by gdziuba**
## Related Issue(s)

close https://github.com/FlowFuse/website/issues/1320

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by gdziuba** [comment]
@ZJvandeWeg When would you like this to go out?

File: src/blog/2023/11/chatgpt-GPT.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Node-Red Builder a GPT (Alpha) by FlowFuse"
+subtitle: "Speed Up Flow Creation with Your Personal Assistant"
+description: "Speed Up Flow Creation with Your Personal Assistant"
+date: 2023-11-15
+authors: ["grey-dziuba"]
+image: "/blog/2023/10/images/buildvbuy.png"
```
Comment (ZJvandeWeg) [inline src/blog/2023/11/chatgpt-GPT.md:7]: New one?

File: src/blog/2023/11/chatgpt-GPT.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    - flowfuse
+
+---
+
+When ChatGPT was first released, my expectations were quite low. I had grown accustomed to the usual industry buzz for AI and ML that often led to underwhelming solutions. Naturally, I approached ChatGPT with similar reservations. It wasn't until a few months after its announcement that I decided to give it a try. To my surprise, within just 10 minutes, I found myself so captivated that I decided to purchase the pro version.
+
+<!--more-->
+
+
+On November 6th, OpenAI unveiled a new offering: GPTs. These function as custom ChatGPT environments, allowing the author to provide additional context, giving it a specific and focused purpose. FlowFuse and Node-RED content go hand in hand, making this a perfect application. With new content emerging daily for both platforms, the ability to update and provide essential documentation became increasingly valuable.
```
Comment (ZJvandeWeg) [inline src/blog/2023/11/chatgpt-GPT.md:21]: I'm not sure what you mean with the second sentence? It goes hand in hand, sure, but why is this the bridge for the rest of the line?

File: src/blog/2023/11/chatgpt-GPT.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+ChatGPT is already a fantastic tool for building Node-RED flows, and if you haven't tried it yet, I highly recommend giving it a go. Now, let me introduce you to Node-Red Builder, a preconfigured environment where all the necessary prompts are already set up to ensure your success. Furthermore, the latest knowledge on Node-RED and FlowFuse is readily available within the GPT, allowing you to tap into the most up-to-date documentation for your prompts.
+
+Node-Red Builder streamlines the development of Node-RED flows, making it more accessible, especially for those new to this environment. We've even provided context to emphasize the use of default nodes over function nodes. Imagine being able to simply drag and drop elements, connect nodes, and create functional flows without delving deep into complex coding. This is precisely what Node-Red Builder makes easier, effectively opening the doors of Node-RED to a wider audience.
+
+[Access to the GPT - Node-RED builder by FlowFuse](https://chat.openai.com/g/g-V5Kyn4omE-node-red-builder-by-flowfuse-v1-0-2)
```
Comment (ZJvandeWeg) [inline src/blog/2023/11/chatgpt-GPT.md:27]:
```suggestion
[Access to the GPT - Node-RED builder by FlowFuse](https://chat.openai.com/g/g-V5Kyn4omE-node-red-builder-by-flowfuse-v1-0-2)

```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by ZJvandeWeg** [comment]
@gdziuba Tomorrow or beyond works for me!

File: src/blog/2023/11/chatgpt-GPT.md (hunk 1 above)
Comment (gdziuba) [reply src/blog/2023/11/chatgpt-GPT.md:7]: Yes.  I made an art request.

File: src/blog/2023/11/chatgpt-GPT.md (hunk 2 above)
Comment (gdziuba) [reply src/blog/2023/11/chatgpt-GPT.md:21]: ChatGPT uses old data.  With the announcement of GPTs, we can add our updated documentation for ChatGPT to reference.  I will see if I can clarify.

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**